spec/fabricators/grpc/activity_task_fabricator.rb

```diff
@@ -0,0 +1,15 @@
+require 'securerandom'
+
+Fabricator(:api_activity_task, from: Temporal::Api::WorkflowService::V1::PollActivityTaskQueueResponse) do
+  transient :task_token, :activity_name, :headers
+
+  activity_id { SecureRandom.uuid }
+  task_token { |attrs| attrs[:task_token] || SecureRandom.uuid }
+  activity_type { Fabricate(:api_activity_type) }
+  input Temporal::Workflow::Serializer::Payload.new('').to_proto
```

Comment:
Not sure about using the serializer in a test, maybe it's fine. Might want to use the verbose Api structure or a test specific helper though?

Reply:
I think nested Fabricator might be a better options here similar to `activity_type`